.travis.yml

```diff
@@ -1,4 +1,7 @@
 language: node_js
+before_install:
+  - curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
This will always install the latest stable version. You might want to lock it down to whatever version you land in #415

Syntax then is `curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version [version]` as per the docs: https://yarnpkg.com/lang/en/docs/install/#alternatives-stable